lib/cli/env.js

```diff
@@ -8,5 +8,15 @@ module.exports = function enrichWithENV(options) {
   options.backendless.apiUri = process.env.APIURI || options.backendless.apiUri
   options.backendless.repoPath = process.env.REPO_PATH || options.backendless.repoPath
 
+  options.backendless.publicHost = process.env.PUBLIC_HOST || options.backendless.publicHost
```

Comment:
I believe we can remove `options.backendless.publicHost/publicPort/publicProtocol ` after composing `options.backendless.public`

Reply:
Removed